Makefile

```diff
@@ -400,6 +404,7 @@ prd/index.html: src/index.mako.html \
 	    ${HTMLMIN_CMD} \
 	    .build-artefacts/last-api-url \
 	    .build-artefacts/last-mapproxy-url \
+	    .build-artefacts/last-shop-url \
```

Comment:
minor and not directly related to this PR but we doing:
```
.build-artefacts/last-api-url \
.build-artefacts/last-mapproxy-url \
.build-artefacts/last-shop-url \
.build-artefacts/last-apache-base-path \
```

6 times, we could maybe factorize this rule.